sky/skylet/ray_patches/__init__.py

```diff
@@ -28,6 +28,7 @@ def _run_patch(target_file, patch_file):
     # .orig is the original file that is not patched.
     orig_file = os.path.abspath(target_file + '.orig')
     script = f"""\
+    which patch >/dev/null 2>&1 || sudo yum install -y patch
```

Comment:
`yum` may not be available on all images. Should we catch this gracefully? E.g., `|| true` at end of L31, and on a new command check `which patch` is available and if not exit early?

Reply:
Ahh, good catch! Just added. Thanks!